content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,206 @@ Calendar
+:guilabel:`Start` section.
+
+The signed-in user will auto-populate as the first attendee. Additional attendees can be added from
+existing contacts or created from the event page.
+
+For virtual meetings, copy and paste the URL into the space provided or select :guilabel:`+Odoo
+meeting` to create a link. Once the event is created, users can click into the virtual meeting
+directly from the calendar event.
+
+Click :guilabel:`More Options` to navigate to the meeting page, which has additional configurations
+for the event:
+
+- :guilabel:`Tags`: Add tags like *Customer Meeting* or *Internal Meeting*. These can be searched
+  and filtered in the Calendar app when organizing multiple events.
+- :guilabel:`Appointment`: Link existing or new appointments that can be configured in
+  :guilabel:`Share Availabilities`.
+- :guilabel:`Privacy`: Toggle between visibility options to control who can view the event.
+- :guilabel:`Organizer`: This is defaulted to the current Odoo user. Select a new Organizer from
+  existing contacts or create and edit a new contact.
+- :guilabel:`Reminders`: Select notification options to sent to attendees. Choose a default
+  notification or configure new reminders.
+
+Coordinate with teams
+---------------------
+
+When scheduling an event for multiple users, tick the checkbox next to :guilabel:`Attendees` to
+view team members' availability. Tick the checkbox next to users to show or hide individual
+calendars.
+
+.. image:: calendar/calendar-attendees.png
+   :align: center
+   :alt: View of Attendees section on Calendar app.
+
+Share Availabilities
+--------------------
+
+Click the :guilabel:`Share Availabilities` box to share external meeting invitations. On the
+calendar, select the target time and date to add them as options in the invitation. Once
+availability has been selected, click the :guilabel:`Open` button to navigate to the associated
+meeting.
+
+.. image:: calendar/calendar-meeting-share-availability.png
+   :align: center
+   :alt: Share availability window on Calendar app.
+
+On the meeting page, users have several configuration options:
+
+Under :guilabel:`Scheduling`, select a minimum hour window to ensure that appointments are
+confirmed a specified amount of time in advance. For example, select :guilabel:`01:00` to require
+attendees to confirm at least one hour before their appointment time.
+
+Under :guilabel:`Allow Cancelling` select a minimum hour window required before the appointment if
+attendees need to make a cancellation.
+
+Select *Users* or *Resources* under the :guilabel:`Availability on` section to allow invitees
+to select internal users to meet or supplies like tables to reserve for their meeting.
+
+The :guilabel:`Availability on` section enables attendees to book *Users* or *Resources* such as
+meeting rooms or tables for their appointment. Select *Users* or *Resources*, then type in the
+contact or resource in the space below.
```

Comment:
I feel like this explanation is better than the previous lines re:Availability on